File: src/main/kotlin/xingkong/loopicserver/picproc/LooFromConfigFile.java
package xingkong.loopicserver.picproc;


import xingkong.loopicserver.picproc.tagImg.ConfigFileManager;

import java.util.Scanner;

public class LooFromConfigFile {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        ConfigFileManager cf = new ConfigFileManager();
        System.out.println("---------------------------------");
        String dir = in.next();
        cf.LooFromConfigFile(dir);

    }
}

File: src/main/kotlin/xingkong/loopicserver/picproc/LooFromConfigFile.kt
package xingkong.loopicserver.picproc


import xingkong.loopicserver.picproc.tagImg.ConfigFileManager
import java.util.*

object LooFromConfigFile {
    @JvmStatic
    fun main(args: Array<String>) {
        val `in` = Scanner(System.`in`)
        val cf = ConfigFileManager()
        println("---------------------------------")
        val dir = `in`.next()
        cf.LooFromConfigFile(dir)

    }
}
